regex-posix.cabal

```diff
@@ -88,5 +98,4 @@ library
   if !impl(ghc >= 8)
       build-depends: fail == 4.9.*
 
-  ghc-options: -O2
```

Comment:
Ditto removal of `-O2`. You cannot even to this in a revision IIRC.

Reply:
Ok, then I am going for releasing 0.96.0.1.